forem/forem
# Prevent whitespace unicode characters from being used in the title

## What type of PR is this? (check all applicable)


## Description
Performs a validation on an article title at the model level and strips out any whitespace unicode characters that would not display in the title or tags

## Related Tickets & Documents
Issue: https://github.com/forem/forem/issues/4969
More characters here for QA purposes: https://jkorpela.fi/chars/spaces.html

## QA Instructions, Screenshots, Recordings

- Login as a user
- Click the "Create Post" button
- Copy U+202D unicode character into the character and tag fields
- Make sure that after you publish the post there is no blank title and blank tag

## Added/updated tests?

      have not been included_

## Are there any post deployment tasks we need to perform?
No, but it's possible that there is bad data on the system in the form of "blank" articles. In a separate pull request I can write a rake task to remediate this or to notify users and ask them to fix these issues if necessary.

## What gif best describes this PR or how it makes you feel?
![Happy gorilla coding](https://user-images.githubusercontent.com/12435007/135745558-cf34bd03-2f08-4a68-b1ad-a9f0424443b3.gif)
